README.md

```diff
@@ -1,83 +1,32 @@
 
-* [CONTRIBUTING.md](CONTRIBUTING.md)
-* [REVIEW_CHECKLIST.md](REVIEW_CHECKLIST.md)
-* [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html)
-* [Ansible Development Guide](https://docs.ansible.com/ansible/devel/dev_guide/index.html)
-* [Ansible Collection Development Guide](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections)
+- [analysis](./roles/analysis/) - executes the Leapp pre-upgrade phase
+- [logging](./roles/logging/) - used for local logging and mutex locking
+- [parse_leapp_report](./roles/parse_leapp_report/) - reads pre-upgrade results and checks for inhibitors
+- [upgrade](./roles/upgrade/) - executes the Leapp OS upgrade
+- [validation](./roles/validation/) - performs post-upgrade validation checks
 
-## Collection maintenance
+## Example playbooks
 
-The current maintainers are listed in the [MAINTAINERS](MAINTAINERS) file. If you have questions or need help, feel free to mention them in the proposals.
+Example playbooks can be found [here](./playbooks/).
 
-To learn how to maintain / become a maintainer of this collection, refer to the [Maintainer guidelines](MAINTAINING.md).
+## Contributing
 
-## Governance
+We are a fledgling community and welcome any new contributors. Get started by opening an issue or pull request. Refer to our [contribution guide](CONTRIBUTING.md) for more information.
 
-<!--Describe how the collection is governed. Here can be the following text:-->
-
-The process of decision making in this collection is based on discussing and finding consensus among participants.
-
-Every voice is important. If you have something on your mind, create an issue or dedicated discussion and let's discuss it!
-
-## Tested with Ansible
-
-<!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
-
-## External requirements
-
-<!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
-
-### Supported connections
-<!-- Optional. If your collection supports only specific connection types (such as HTTPAPI, netconf, or others), list them here. -->
-
-## Included content
-
-<!-- Galaxy will eventually list the module docs within the UI, but until that is ready, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
-
-## Using this collection
-
-<!--Include some quick examples that cover the most common use cases for your collection content. It can include the following examples of installation and upgrade (change infra.leapp correspondingly):-->
-
-### Installing the Collection from Ansible Galaxy
+## Installing the collection from Ansible Galaxy
 
 Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
```

Comment:
@heatmiser @djdanielsson @jeffmcutter:
Question for you Ansible gurus. This whole "Installing the collection from Ansible Galaxy" section is copied from the original README boilerplate. Just curious if these commands are relevant or if we just need to give the requirements.yml example assuming people will be using the collection from AAP?

Reply:
As discussed in the meeting, we'll leave this as is.